lib/compiler/src/beam_ssa.erl

```diff
@@ -404,26 +404,24 @@ common_dominators(Ls, Dom, Numbering) ->
     Doms = [map_get(L, Dom) || L <- Ls],
     dom_intersection(Doms, Numbering).
 
--spec fold_instrs_rpo(Fun, From, Acc0, Blocks) -> any() when
+-spec fold_instrs_rpo(Fun, Labels, Acc0, Blocks) -> any() when
       Fun :: fun((b_blk()|terminator(), any()) -> any()),
-      From :: [label()],
+      Labels :: [label()],
       Acc0 :: any(),
       Blocks :: block_map().
 
-fold_instrs_rpo(Fun, From, Acc0, Blocks) ->
-    Top = rpo(From, Blocks),
+fold_instrs_rpo(Fun, Top, Acc0, Blocks) ->
     fold_instrs_rpo_1(Top, Fun, Blocks, Acc0).
 
 %% Like mapfold_instrs_rpo but at the block level to support lookahead and
 %% scope-dependent transformations.
--spec mapfold_blocks_rpo(Fun, From, Acc, Blocks) -> Result when
+-spec mapfold_blocks_rpo(Fun, Labels, Acc, Blocks) -> Result when
```

Comment:
I suggest you rename this function to `mapfold_blocks`. Also rewrite the comment before to a comment in the proper style.